app/views/questionnaire/secondpage.scala.html

```diff
@@ -47,8 +46,18 @@ <h2 class="heading-medium">University</h2>
                     @university(questionnaireForm)
                     @universityDegreeCategory(questionnaireForm)
                 </div>
-            }
-        </fieldset>
+            </fieldset>
+        } else {
+            <h2 class="heading-medium">University</h2>
+            <div class="panel-indent text">
+                <p>This data helps us compare the average performance of certain universities
+                    to the national average.</p>
+            </div>
+            <input type="hidden" id="haveDegree" name="haveDegree" value="Yes">
+            @university(questionnaireForm)
```

Comment:
Shall we move the following templates
@university(questionnaireForm)
@universityDegreeCategory(questionnaireForm)
outside from if/else